src/jwt.ts

```diff
@@ -0,0 +1,12 @@
+import jwt from 'jsonwebtoken'
+
+import {JsonWebTokenConfig, RequestData} from './types'
+
+const verifyJWT = (config: JsonWebTokenConfig) => {
+  jwt.verify(config.token, config.secretOrPublicKey, config.options)
+}
+
+const generateJWT = (data: RequestData, secretOrPrivateKey: jwt.Secret, options?: jwt.SignOptions) =>
+  jwt.sign(data, secretOrPrivateKey, options)
```

Comment:
This won't necessarily support secp256k1 or custom libsodium or otherwise experiemental JWS formats.

Its clear from the tests how to inject your own custom JWT/JWS, but I'm not sure its possible to inject a verify function. 

You would need to do this for secp256k1 or ethereumAddress based verification.